### DISCUSSION TITLE
Default class overflow-hidden blocking dropdown field

### DISCUSSION THREAD

**Discussion by bocanhcam** [laravel/nova-issues#6387] [Q&A] [answered]
I create a custom date field and the parent's div in the resource which has `overflow-hidden`  class blocks the date.
![Screenshot from 2024-05-15 15-11-14](https://github.com/laravel/nova-issues/assets/35592955/abbf6cc4-8c26-420e-9ee4-efcf2255974f)

- Laravel Version: 10
- Nova Version: 4

**Comment by crynobone** [comment, marked as answer]
This seems to be a 3rd party package where it should use teleport to append to body instead of within the container.
